index.js

```diff
@@ -145,32 +168,33 @@ module.exports = {
     // Prefer destructuring from arrays and objects
     // https://eslint.org/docs/rules/prefer-destructuring
     // We turn this off for assignments.
-    'prefer-destructuring': ['error', {
-      VariableDeclarator: {
-        array: false,
-        object: true,
+    'prefer-destructuring': [
+      'error',
+      {
+        VariableDeclarator: {
+          array: false,
+          object: true,
+        },
+        AssignmentExpression: {
+          array: false,
+          object: false,
+        },
       },
-      AssignmentExpression: {
-        array: false,
-        object: false,
+      {
+        enforceForRenamedProperties: false,
       },
-    }, {
-      enforceForRenamedProperties: false,
-    }],
+    ],
 
     // specify whether double or single quotes should be used. Allows template literals.
-    quotes: ['error', 'single', { avoidEscape: true, allowTemplateLiterals: true }],
+    quotes: [
+      'error',
+      'single',
+      { avoidEscape: true, allowTemplateLiterals: true },
+    ],
 
     // semi-colons should always be at the end of statements.
     'semi-style': ['error', 'last'],
 
-    // Don't use spaces before function parens except when using "await".
```

Comment:
This isn't needed because `prettier` disables this rule anyway. It was not a problem before because each project required `prettier` after the `amo` config so `prettier` config/plugin disabled the rule. Now, it is a bit different: we load the prettier config/plugin fist in this config.